docker/Dockerfile.bionic-glibc

```diff
@@ -1,25 +1,34 @@
-FROM ubuntu:bionic
+ARG DISTRO=bionic
 
+FROM ubuntu:${DISTRO}
```

Comment:
Now that the file supports both bionic and xenial, maybe just rename file to Dockerfile.ubuntu-glibc WDYT @alexeldeib